mlflow/store/tracking/abstract_store.py

```diff
@@ -239,28 +239,50 @@ def restore_run(self, run_id):
         """
         pass
 
-    def create_trace_info(
+    def start_trace(
         self,
         experiment_id: str,
         timestamp_ms: int,
```

Comment:
Why do we remove `status` in start_trace?

Reply:
We only know the status once the trace is ended